scripts/commands.py

```diff
@@ -0,0 +1,42 @@
+"""
+This module defines the Commands class which provides an interface for executing SQL commands.
+
+TODO:
+    * Add more command methods as needed.
+    * Store temporary informations in a file
+"""
+
+from scripts.db_query_handler import DBQueryHandler
+from scripts.utils.json import get_key_list
+
+
+class Commands:
+    """
+    A class for executing database commands.
+
+    Attributes:
+        handler (DBQueryHandler): Executes the queries.
+    """
+
+    def __init__(self, handler: DBQueryHandler, config: dict):
```

Comment:
`dict` is ok as type, but optionally you could define a better type that shows also the properties of the dict

Reply:
Currently the config is empty for commands section